.github/workflows/repository-scanner.yml

```diff
@@ -0,0 +1,29 @@
+name: Scan Actions
```

Comment:
Is this version only run in the local repository? I'm trying to understand all of the different yml files I find here.